semantic-python/test-graphing/GraphTest.hs

```diff
@@ -50,10 +51,10 @@ The graph should be
 runScopeGraph :: ToScopeGraph t => Path.AbsRelFile -> Source.Source -> t Loc -> (ScopeGraph.ScopeGraph Name, Result)
 runScopeGraph p _src item = run . runSketch (Just p) $ scopeGraph item
 
-sampleGraphThing :: (Has (Sketch Name) sig m) => m Result
+sampleGraphThing :: (Has Sketch sig m) => m Result
 sampleGraphThing = do
-  declare @Name "hello" DeclProperties
-  declare @Name "goodbye" DeclProperties
+  declare "hello" (DeclProperties ScopeGraph.Assignment ScopeGraph.Default Nothing)
```

Comment:
Now that `DeclProperties` is being used for something useful, should we maybe give it its own module?